### PR TITLE
Merge 9.0.x into develop

### PR DESCRIPTION
**Pull request by tleon**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      | Merge 9.0.x into develop
| Type?             |  improvement 
| Category?         |  ME 
| BC breaks?        |  no
| Deprecations?     |no
| How to test?      | Ci & tests green
| UI Tests          | https://github.com/tleon/ga.tests.ui.pr/actions/runs/11937482179
| Sponsor company   | PrestaShop SA

### CODE REVIEW

**Review by nesrineabdmouleh — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]
OK for ui TEsts

**Review by boherm — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
